src/drivers/storage/SDCard.h

```diff
@@ -58,8 +58,13 @@ class SDCard
     bool loadConfigFile(TSettings* Settings);
     bool cardAvailable();
     bool cardBusy();
+#ifdef NERDMINER_T_HMI
```

Comment:
Is this code only related to T-HMI or is there a more generic name for this condition ?

Reply:
just being careful not to eff up other builds and cause a regression.  it would have been easier to make that method public in general.